fineract-provider/src/main/java/org/apache/fineract/commands/service/SynchronousCommandProcessingService.java

```diff
@@ -125,7 +143,8 @@ public CommandProcessingResult processAndLogCommand(final CommandWrapper wrapper
         }
         result.setRollbackTransaction(null);
 
-        publishEvent(wrapper.entityName(), wrapper.actionName(), result);
+        // publishEvent(wrapper.entityName(), wrapper.actionName(), result);
+        publishEvent(wrapper.entityName(), wrapper.actionName(), command, result);
```

Comment:
Just removing such lines entirely instead of leaving them commented out like this is less "cluttered" (there is always git, for history):

```suggestion
        publishEvent(wrapper.entityName(), wrapper.actionName(), command, result);
```

Reply:
corrected!